packages/office-addin-debugging/src/commands.ts

```diff
@@ -34,29 +75,34 @@ export async function start(manifestPath: string, appType: string | undefined, c
         const devServerPort = parseDevServerPort(command.devServerPort || process.env.npm_package_config_dev_server_port);
         const enableDebugging: boolean = command.debug;
         const enableLiveReload: boolean = (command.liveReload === true);
+        const manifestPathToDebug: string = getManifestPath(manifestPath, appTypeToDebug, command.prod);
         const packager: string | undefined = command.packager || process.env.npm_package_scripts_packager;
         const packagerHost: string | undefined = command.PackagerHost || process.env.npm_package_config_packager_host;
         const packagerPort: string | undefined = command.PackagerPort || process.env.npm_package_config_packager_port;
         const sideload: string | undefined = command.sideload || process.env[`npm_package_scripts_sideload_${app}`] || process.env.npm_package_scripts_sideload;
+        const sideloadCommand: string | undefined = getSideloadCommand(sideload, manifestPathToDebug);
 
         if (appTypeToDebug === undefined) {
             throw new Error("Please specify the application type to debug.");
         }
 
-        await startDebugging(manifestPath, appTypeToDebug, debuggingMethod, sourceBundleUrlComponents,
-            devServer, devServerPort, packager, packagerHost, packagerPort, sideload,
+        await startDebugging(manifestPathToDebug, appTypeToDebug, debuggingMethod, sourceBundleUrlComponents,
+            devServer, devServerPort, packager, packagerHost, packagerPort, sideloadCommand,
             enableDebugging, enableLiveReload);
     } catch (err) {
         logErrorMessage(`Unable to start debugging.\n${err}`);
     }
 }
 
-export async function stop(manifestPath: string, command: commander.Command) {
+export async function stop(manifestPath: string, appType: string | undefined, command: commander.Command) {
```

Comment:
The appType parameter should not allow undefined. It looks like this function never handles it when it is undefined.